### DISCUSSION TITLE
Is the version provided by Web Flasher relatively stable?

### DISCUSSION THREAD

**Discussion by Mrz-creator** [darkxst/silabs-firmware-builder#128] [Q&A] [answered]
You mentioned "however, the builds offered via the Web Flasher are the latest versions recommended by the community." in the README. Can I think that the firmware provided by Web Flasher is relatively stable? I would also like to ask you to point me to the "recommended community". Is it HA?

**Comment by darkxst** [comment, marked as answer]
The builds in the web flasher only get updated after testings and sufficient feedback from users. Thus they generally lag behind the latest builds available in repo. 

For communities, yes HA and Z2M

**Reply by Mrz-creator** [reply]
OK, thanks！
